test/integration/buoyancy_engine.cc

```diff
@@ -0,0 +1,149 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <gtest/gtest.h>
+
+#include <ignition/common/Console.hh>
+#include <ignition/common/Util.hh>
+#include <ignition/msgs/double.pb.h>
+#include <ignition/transport/Node.hh>
+
+#include "ignition/gazebo/Server.hh"
+#include "ignition/gazebo/SystemLoader.hh"
+#include "ignition/gazebo/components/Link.hh"
+#include "ignition/gazebo/components/Name.hh"
+#include "ignition/gazebo/components/Pose.hh"
+#include "ignition/gazebo/components/Model.hh"
+
+
+#include "ignition/gazebo/test_config.hh"
+#include "../helpers/Relay.hh"
+
+using namespace ignition;
+using namespace gazebo;
+
+class BuoyancyEngineTest : public ::testing::Test
```

Comment:
Since #971 we should use the following for all tests

```suggestion
class BuoyancyEngineTest : public InternalFixture<:::testing::Test>
```